uint8 status
uint64 elapsed_time
float64[] encoder_position
float64[] encoder_speed
float64[] loadcell_torque
float64[] setpoint_torque


